lib/srv/sess.go

```diff
@@ -797,7 +927,7 @@ func (s *session) startInteractive(ch ssh.Channel, ctx *ServerContext) error {
 	go func() {
 		defer s.term.AddParty(-1)
 
-		_, err := io.Copy(s.writer, s.term.PTY())
+		_, err := io.Copy(s.io, s.term.PTY())
 		s.log.Debugf("Copying from PTY to writer completed with error %v.", err)
```

Comment:
Our debug logs are super noisy. Is this debug output useful if err is nil? If not, can we only log if there was an error?

Reply:
I've found it useful on occasion when debugging issues so my preference is to leave it in.